github-notifications/json_payloads.py

```diff
@@ -0,0 +1,593 @@
+test_branch = {
```

Comment:
I feel a little torn about storing this. On the one hand, it makes it way easier to test, on the other, it feels weird to store sample JSON this way. @bfreeds, curious about your thoughts on this. Any good middle ground we could find for having a sample response to test with?

It's less automatic, but I was thinking maybe a note in the file about where to find sample data like in Settings > Webhooks of our repos. No guarantee that it'd be there tho I guess, depending on the repo.

Reply:
Ohh good Q!  I'm definitely no expert on testing, but in the past I've seen static jsons be stored for simulating API responses.  In this case, I'm a fan of this approach.  The main tradeoff I could see is if github changed the api and we didn't realize, then this might work locally and then not after deploying.

Reply:
The other tradeoff would be if there is sensitive data stored in that API response JSON.  The only thing I see that I wondered about is the `id` fields - I couldn't find anything that indicated they are sensitive, and think it's okay.

Reply:
Ok sounds good! Yeah it does make testing much easier to store them. Fine to keep this as is then